store/src/java/com/zimbra/cs/lmtpserver/ExternalEmailWarning.java

```diff
@@ -36,16 +40,17 @@
 public class ExternalEmailWarning {
 
     private static ExternalEmailWarning instance;
-    private boolean isEnabled;
     private String baseWarningMessage;
     private String textPlainWarning;
     private String textHtmlWarning;
     private String encodedTextPlainWarning;
     private String encodedTextHtmlWarning;
 
     private ExternalEmailWarning() {
-        isEnabled = LC.zimbra_external_email_warning_enabled.booleanValue();
-        baseWarningMessage = LC.zimbra_external_email_warning_message.value();
+        try {
+            baseWarningMessage = Provisioning.getInstance().getConfig().getExternalEmailWarningMessage();
+        } catch (ServiceException e) {
```

Comment:
should we add a log entry if we are not able to read ldap contents? would help in debugging